mtp_cashbook/assets-src/javascripts/modules/running-total.js

```diff
@@ -0,0 +1,67 @@
+// Running Total module
+/* globals require, exports */
+'use strict';
```

Comment:
lint says `Use the function form of "use strict".` – but the node was is to have it at top level, right?